src/main/java/mate/academy/dao/MovieDaoImpl.java

```diff
@@ -24,9 +24,9 @@ public Movie add(Movie movie) {
         } catch (HibernateException e) {
             if (transaction != null) {
                 transaction.rollback();
-                throw new DataProcessingException(
-                        "Can't commit transaction. Transaction is rolled back", e);
             }
+            throw new DataProcessingException(
+                    "Can't add '" + movie.getTitle() + "' movie. Transaction is rolled back", e);
```

Comment:
```suggestion
                    "Can't add '" + movie + "' movie to DB", e);
```
якщо прокидування помилки стоїться після if то транзакція могла відкотитися, а могла й ні